M2/Macaulay2/packages/MultigradedImplicitization.m2

```diff
@@ -23,10 +26,11 @@ export {
   -- Methods
   "maxGrading",
   "trimBasisInDegree",
-  "componentOfKernel",
+  "computeComponent",
+  "interpolateComponent",
   "componentsOfKernel",
   -- Options
-  "Grading", "PreviousGens", "ReturnTargetGrading", "UseMatroidSpeedup"
+  "Grading", "PreviousGens", "ReturnTargetGrading", "UseMatroid", "UseInterpolation", "Verbose"
```

Comment:
`Verbose` is already exported by Core, so you shouldn't need to export it here

Reply:
@bkholler I will make this change and push to your repository. Since it's just an update, I think we can include this PR in the June 1st patch release.